torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -887,7 +888,63 @@ RegisterOperators reg(
          userObj->setSlot(slot, std::move(v));
          return 0;
        };
-     })});
+     })
+     });
+
+RegisterOperators logging_operators({
+    Operator("prim::BumpCounter(str key, int val) -> ()", [](Stack& stack) {
+          auto val = pop(stack).toInt();
+          auto key = pop(stack).toString();
+
+          auto schema = parseSchema("prim::BumpCounter(str key, int val) -> ()");
+          if (jit::tracer::isTracing()) {
```

Comment:
I don't like the fact that we have to manually write tracing code for these. This is because of a bug in the custom API that @smessmer has a PR with a fix out for. Can we make sure to update this when the fix lands?

Reply:
Can you User torch::RegisterOperators instead of torch::jit::RegisterOperators? It doesn't give you a Stack but concrete types and you don't need to do tracing manually. Or is this a special case that wouldn't work with that?